GameServerLib/Logic/API/ApiEventManager.cs

```diff
@@ -72,8 +72,12 @@ public static void removeAllListenersForOwner(object owner)
         }
         
         public static EventOnUpdate OnUpdate = new EventOnUpdate();
+        public static EventOnLevelUpSpell OnLevelUpSpell = new EventOnLevelUpSpell();
         public static EventOnChampionDamageTaken OnChampionDamageTaken = new EventOnChampionDamageTaken();
+        public static EventOnChampionDamageDealt OnChampionDamageDealt = new EventOnChampionDamageDealt();
         public static EventOnUnitDamageTaken OnUnitDamageTaken = new EventOnUnitDamageTaken();
+        public static EventOnUnitDamageDealt OnUnitDamageDealt = new EventOnUnitDamageDealt();
+        public static EventOnAutoAttackHit OnAutoAttackHit = new EventOnAutoAttackHit();
```

Comment:
All these listeners you add correspond to non-existing events, here's a list of events (might be out-of-date) https://hastebin.com/xeyavarolo.txt